### PR TITLE
MPDX-7152-Contact-and-Tasks-Show-Featured-Filters

### PR DESCRIPTION
**Pull request by TheNoodleMoose**
I did a couple of different things in this pr:

1. Add logic to show featured filters and hide the rest in the "See more filters" section.
Screen Shot 2021-10-01 at 11 31 01 AM

2. Fixed some logic that was causing selected filters not to show up outside of the "See more filters" section.
Screen Shot 2021-10-01 at 11 32 48 AM

3. Added logic to show the number of selected options per filter.
Screen Shot 2021-10-01 at 11 30 30 AM

4. Fixed the clear all logic as it wasn't updating the filters and results correctly.

I'm not sure if the currently featured filters are the ones that are actually wanted. I didn't see those specified in Figma, but maybe I missed it 🤷‍♂️

### CODE REVIEW

File: src/components/Contacts/ContactFilters/ContactFilters.test.tsx [hunk 1]
```diff
@@ -47,6 +53,24 @@ describe('ContactFilters', () => {
...
+          ContactFiltersDefaultMock.accountList.contactFilterGroups[0].name,
+        ),
+      ).not.toBeVisible(),
+    );
+
+    expect(
+      getByText(
+        ContactFiltersDefaultMock.accountList.contactFilterGroups[1].name,
+      ),
+    ).toBeVisible();
```
Comment (TheNoodleMoose) [inline src/components/Contacts/ContactFilters/ContactFilters.test.tsx:73]: I thought about making a separate test for the featured logic but just went with this. Let me know what you guys think.

**Comment by TheNoodleMoose** [comment]
@OzzieOrca Looks like the random test failure I mentioned happened again. Locally everything passed. It's always in ```chunk 0``` as well.

**Comment by OzzieOrca** [comment]
https://jira.cru.org/browse/MPDX-7161 will update which filters are featured in the API (it's linked to this Jira ticket). There could be some refactoring that comes from it as a result.

Eventually we'll need to bring tags and saved filters to the featured section too but we'll wait to see if the API can implement them in the filter options.

**Review by OzzieOrca — APPROVED** [review]

**Comment by OzzieOrca** [comment]
My guess from reading https://stackoverflow.com/a/52645528/665224 is that one of the `console.error(...)`s in the logs is somehow throwing a top level error after Jest finishes.

Looking through the logs, most of the `console.error` messages are actually React warnings. While we should probably fix them (there are a lot) they probably aren't actually throwing errors.

The following errors stood out. I guess we should create Jira tickets for them.

```
2021-10-01T15:26:52.5374925Z PASS src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonModal.test.tsx (52.24 s)
2021-10-01T15:26:52.5388575Z   ● Console
2021-10-01T15:26:52.5389102Z 
2021-10-01T15:26:52.6364561Z     console.error
2021-10-01T15:26:52.6365304Z       Missing field 'user' while writing result {
2021-10-01T15:26:52.6365834Z         "id": "321",
2021-10-01T15:26:52.6366273Z         "name": "Person, Test",
2021-10-01T15:26:52.6366738Z         "tagList": [
2021-10-01T15:26:52.6367149Z           "tag1",
2021-10-01T15:26:52.6367516Z           "tag2",
2021-10-01T15:26:52.6367910Z           "tag3"
2021-10-01T15:26:52.6368256Z         ],
2021-10-01T15:26:52.6368662Z         "addresses": {
2021-10-01T15:26:52.6369251Z           "nodes": [
2021-10-01T15:26:52.6369643Z             {
2021-10-01T15:26:52.6370287Z               "id": "addres-1",
2021-10-01T15:26:52.6370967Z               "street": "123 Sesame Street",
2021-10-01T15:26:52.6371496Z               "city": "New York",
2021-10-01T15:26:52.6371933Z               "state": "NY",
2021-10-01T15:26:52.6372422Z               "postalCode": "10001",
2021-10-01T15:26:52.6372891Z               "country": "USA",
2021-10-01T15:26:52.6373513Z               "primaryMailingAddress": true,
2021-10-01T15:26:52.6374138Z               "historic": false,
2021-10-01T15:26:52.6374637Z               "region": "some region",
2021-10-01T15:26:52.6375169Z               "source": "some source"
2021-10-01T15:26:52.6375577Z             }
2021-10-01T15:26:52.6375934Z           ]
2021-10-01T15:26:52.6376286Z         },
2021-10-01T15:26:52.6376658Z         "people": {
2021-10-01T15:26:52.6377320Z           "nodes": [
2021-10-01T15:26:52.6377681Z             {
2021-10-01T15:26:52.6378073Z               "id": "1258178",
2021-10-01T15:26:52.6378520Z               "almaMater": "Train",
2021-10-01T15:26:52.6379069Z               "anniversaryDay": 1,
2021-10-01T15:26:52.6379771Z               "anniversaryMonth": 1,
2021-10-01T15:26:52.6380350Z               "anniversaryYear": 1990,
2021-10-01T15:26:52.6380897Z               "birthdayDay": 1,
2021-10-01T15:26:52.6381396Z               "birthdayMonth": 1,
2021-10-01T15:26:52.6381928Z               "birthdayYear": 1990,
2021-10-01T15:26:52.6382422Z               "deceased": true,
2021-10-01T15:26:52.6382958Z               "emailAddresses": {
2021-10-01T15:26:52.6383447Z                 "nodes": [
2021-10-01T15:26:52.6383820Z                   {
2021-10-01T15:26:52.6384227Z                     "id": "6660174",
2021-10-01T15:26:52.6384760Z                     "email": "test1234@test.com",
2021-10-01T15:26:52.6385284Z                     "primary": true,
2021-10-01T15:26:52.6385796Z                     "location": "Work"
2021-10-01T15:26:52.6386109Z 
2021-10-01T15:26:52.6387688Z       at Function.error (.yarn/cache/ts-invariant-npm-0.9.3-7a021097a2-c3f21931f9.zip/node_modules/ts-invariant/lib/invariant.esm.js:35:27)
2021-10-01T15:26:52.6390419Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:149:42
2021-10-01T15:26:52.6392026Z           at Set.forEach (<anonymous>)
2021-10-01T15:26:52.6394638Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
2021-10-01T15:26:52.6398727Z       at StoreWriter.Object.<anonymous>.StoreWriter.processFieldValue (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:192:21)
2021-10-01T15:26:52.6402225Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:123:47
2021-10-01T15:26:52.6403817Z           at Set.forEach (<anonymous>)
2021-10-01T15:26:52.6406435Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
```

```
2021-10-01T15:27:35.1815127Z PASS src/components/Layouts/Primary/TopBar/Items/NotificationMenu/Item/Item.test.tsx
2021-10-01T15:27:35.1816332Z   ● Console
2021-10-01T15:27:35.1816611Z 
2021-10-01T15:27:35.1852776Z     console.error
2021-10-01T15:27:35.1853349Z       Error: Not implemented: window.open
2021-10-01T15:27:35.1855505Z           at module.exports (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
2021-10-01T15:27:35.1858257Z           at open (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/browser/Window.js:728:7)
2021-10-01T15:27:35.1860230Z           at onClick (/home/runner/work/mpdx-react/mpdx-react/src/components/HandoffLink/HandoffLink.tsx:39:16)
2021-10-01T15:27:35.1862992Z           at HTMLUnknownElement.callCallback (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:3945:14)
2021-10-01T15:27:35.1866960Z           at HTMLUnknownElement.callTheUserObjectsOperation (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
2021-10-01T15:27:35.1870648Z           at innerInvokeEventListeners (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:318:25)
2021-10-01T15:27:35.1873891Z           at invokeEventListeners (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
2021-10-01T15:27:35.1877157Z           at HTMLUnknownElementImpl._dispatch (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
2021-10-01T15:27:35.1880659Z           at HTMLUnknownElementImpl.dispatchEvent (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
2021-10-01T15:27:35.1884290Z           at HTMLUnknownElement.dispatchEvent (/home/runner/work/mpdx-react/mpdx-react/.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34) undefined
2021-10-01T15:27:35.1885854Z 
2021-10-01T15:27:35.1886188Z       37 |       }
2021-10-01T15:27:35.1886719Z       38 |       if (!e.defaultPrevented) {
2021-10-01T15:27:35.1887531Z     > 39 |         window.open(url.href, '_blank');
2021-10-01T15:27:35.1888036Z          |                ^
2021-10-01T15:27:35.1888521Z       40 |         e.preventDefault();
2021-10-01T15:27:35.1889002Z       41 |       }
2021-10-01T15:27:35.1889362Z       42 |     },
2021-10-01T15:27:35.1889590Z 
2021-10-01T15:27:35.1891433Z       at VirtualConsole.<anonymous> (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/virtual-console.js:29:45)
2021-10-01T15:27:35.1894045Z       at module.exports (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:12:26)
2021-10-01T15:27:35.1896400Z       at open (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/browser/Window.js:728:7)
2021-10-01T15:27:35.1897966Z       at onClick (src/components/HandoffLink/HandoffLink.tsx:39:16)
2021-10-01T15:27:35.1900357Z       at HTMLUnknownElement.callCallback (.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:3945:14)
2021-10-01T15:27:35.1904045Z       at HTMLUnknownElement.callTheUserObjectsOperation (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
2021-10-01T15:27:35.1907343Z       at innerInvokeEventListeners (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:318:25)
2021-10-01T15:27:35.1908634Z 
```
```
2021-10-01T15:28:04.3739991Z PASS src/components/Task/Drawer/CompleteForm/CompleteForm.test.tsx (29.259 s)
2021-10-01T15:28:04.4547172Z   ● Console
2021-10-01T15:28:04.4548165Z 
2021-10-01T15:28:04.5018823Z     console.error
2021-10-01T15:28:04.5019900Z       Material-UI: A component is changing the uncontrolled value state of Select to be controlled.
2021-10-01T15:28:04.5020977Z       Elements should not switch from uncontrolled to controlled (or vice versa).
2021-10-01T15:28:04.5022086Z       Decide between using a controlled or uncontrolled Select element for the lifetime of the component.
2021-10-01T15:28:04.5023632Z       The nature of the state is determined during the first render, it's considered controlled if the value is not `undefined`.
2021-10-01T15:28:04.5025064Z       More info: https://fb.me/react-controlled-components
2021-10-01T15:28:04.5025645Z 
2021-10-01T15:28:04.5027676Z       at .yarn/$$virtual/@material-ui-core-virtual-9fb4a16d1c/0/cache/@material-ui-core-npm-4.11.3-1281c40e43-1bca52d08e.zip/node_modules/@material-ui/core/utils/useControlled.js:32:17
2021-10-01T15:28:04.5030972Z       at invokePassiveEffectCreate (.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:23487:20)
2021-10-01T15:28:04.5034323Z       at HTMLUnknownElement.callCallback (.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:3945:14)
2021-10-01T15:28:04.5037947Z       at HTMLUnknownElement.callTheUserObjectsOperation (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
2021-10-01T15:28:04.5041318Z       at innerInvokeEventListeners (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:318:25)
2021-10-01T15:28:04.5044227Z       at invokeEventListeners (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
2021-10-01T15:28:04.5047131Z       at HTMLUnknownElementImpl._dispatch (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
2021-10-01T15:28:04.5050309Z       at HTMLUnknownElementImpl.dispatchEvent (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
2021-10-01T15:28:04.5052006Z 
```
```
2021-10-01T15:29:32.6936818Z PASS src/components/Task/Drawer/Form/Form.test.tsx (13.427 s)
2021-10-01T15:29:32.8312838Z   ● Console
2021-10-01T15:29:39.7533637Z     console.error
2021-10-01T15:29:39.7534845Z       Material-UI: A component is changing the uncontrolled value state of Autocomplete to be controlled.
2021-10-01T15:29:39.7535951Z       Elements should not switch from uncontrolled to controlled (or vice versa).
2021-10-01T15:29:39.7537117Z       Decide between using a controlled or uncontrolled Autocomplete element for the lifetime of the component.
2021-10-01T15:29:39.7538701Z       The nature of the state is determined during the first render, it's considered controlled if the value is not `undefined`.
2021-10-01T15:29:39.7540056Z       More info: https://fb.me/react-controlled-components
2021-10-01T15:29:39.7540630Z 
2021-10-01T15:29:39.7542683Z       at .yarn/$$virtual/@material-ui-core-virtual-9fb4a16d1c/0/cache/@material-ui-core-npm-4.11.3-1281c40e43-1bca52d08e.zip/node_modules/@material-ui/core/utils/useControlled.js:32:17
2021-10-01T15:29:39.7545964Z       at invokePassiveEffectCreate (.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:23487:20)
2021-10-01T15:29:39.7549321Z       at HTMLUnknownElement.callCallback (.yarn/$$virtual/react-dom-virtual-285e68f0be/0/cache/react-dom-npm-17.0.1-588d0088ca-6a70028fbe.zip/node_modules/react-dom/cjs/react-dom.development.js:3945:14)
2021-10-01T15:29:39.7553264Z       at HTMLUnknownElement.callTheUserObjectsOperation (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
2021-10-01T15:29:39.7556986Z       at innerInvokeEventListeners (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:318:25)
2021-10-01T15:29:39.7560799Z       at invokeEventListeners (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
2021-10-01T15:29:39.7563757Z       at HTMLUnknownElementImpl._dispatch (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
2021-10-01T15:29:39.7566924Z       at HTMLUnknownElementImpl.dispatchEvent (.yarn/$$virtual/jsdom-virtual-89425ac844/0/cache/jsdom-npm-16.4.0-2745a1cb43-adca681df0.zip/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
2021-10-01T15:29:39.7568737Z 
2021-10-01T15:29:39.7946825Z     console.error
2021-10-01T15:29:39.7947808Z       Missing field 'totalCount' while writing result {
2021-10-01T15:29:39.7948815Z         "nodes": [
2021-10-01T15:29:39.7949184Z           {
2021-10-01T15:29:39.7949648Z             "activityType": null,
2021-10-01T15:29:39.7950124Z             "contacts": {
2021-10-01T15:29:39.7950549Z               "nodes": []
2021-10-01T15:29:39.7950910Z             },
2021-10-01T15:29:39.7951587Z             "id": "task-1",
2021-10-01T15:29:39.7952218Z             "notificationTimeBefore": null,
2021-10-01T15:29:39.7952945Z             "notificationTimeUnit": null,
2021-10-01T15:29:39.7953607Z             "notificationType": null,
2021-10-01T15:29:39.7954378Z             "startAt": "2013-01-05T01:02:00.000+00:00",
2021-10-01T15:29:39.7955125Z             "completedAt": "2016-01-05T01:02:00.000+00:00",
2021-10-01T15:29:39.7955633Z             "subject": "",
2021-10-01T15:29:39.7956273Z             "tagList": [],
2021-10-01T15:29:39.7956705Z             "user": null
2021-10-01T15:29:39.7957069Z           }
2021-10-01T15:29:39.7957411Z         ]
2021-10-01T15:29:39.7957731Z       }
2021-10-01T15:29:39.7957968Z 
2021-10-01T15:29:39.7959550Z       at Function.error (.yarn/cache/ts-invariant-npm-0.9.3-7a021097a2-c3f21931f9.zip/node_modules/ts-invariant/lib/invariant.esm.js:35:27)
2021-10-01T15:29:39.7962466Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:149:42
2021-10-01T15:29:39.7964119Z           at Set.forEach (<anonymous>)
2021-10-01T15:29:39.7966803Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
2021-10-01T15:29:39.7971106Z       at StoreWriter.Object.<anonymous>.StoreWriter.processFieldValue (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:192:21)
2021-10-01T15:29:39.7974717Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:123:47
2021-10-01T15:29:39.7976352Z           at Set.forEach (<anonymous>)
2021-10-01T15:29:39.7979009Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
2021-10-01T15:29:39.7980897Z 
2021-10-01T15:29:39.7981316Z     console.error
2021-10-01T15:29:39.7982115Z       Missing field 'pageInfo' while writing result {
2021-10-01T15:29:39.7982656Z         "nodes": [
2021-10-01T15:29:39.7983027Z           {
2021-10-01T15:29:39.7983468Z             "activityType": null,
2021-10-01T15:29:39.7983960Z             "contacts": {
2021-10-01T15:29:39.7984371Z               "nodes": []
2021-10-01T15:29:39.7984748Z             },
2021-10-01T15:29:39.7985294Z             "id": "task-1",
2021-10-01T15:29:39.7985899Z             "notificationTimeBefore": null,
2021-10-01T15:29:39.7986643Z             "notificationTimeUnit": null,
2021-10-01T15:29:39.7987294Z             "notificationType": null,
2021-10-01T15:29:39.7988321Z             "startAt": "2013-01-05T01:02:00.000+00:00",
2021-10-01T15:29:39.7989069Z             "completedAt": "2016-01-05T01:02:00.000+00:00",
2021-10-01T15:29:39.7989562Z             "subject": "",
2021-10-01T15:29:39.7989997Z             "tagList": [],
2021-10-01T15:29:39.7990536Z             "user": null
2021-10-01T15:29:39.7990914Z           }
2021-10-01T15:29:39.7991243Z         ]
2021-10-01T15:29:39.7991582Z       }
2021-10-01T15:29:39.7991813Z 
2021-10-01T15:29:39.7993348Z       at Function.error (.yarn/cache/ts-invariant-npm-0.9.3-7a021097a2-c3f21931f9.zip/node_modules/ts-invariant/lib/invariant.esm.js:35:27)
2021-10-01T15:29:39.7996088Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:149:42
2021-10-01T15:29:39.7997726Z           at Set.forEach (<anonymous>)
2021-10-01T15:29:39.8000371Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
2021-10-01T15:29:39.8004522Z       at StoreWriter.Object.<anonymous>.StoreWriter.processFieldValue (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:192:21)
2021-10-01T15:29:39.8008079Z       at .yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:123:47
2021-10-01T15:29:39.8009696Z           at Set.forEach (<anonymous>)
2021-10-01T15:29:39.8012522Z       at StoreWriter.Object.<anonymous>.StoreWriter.processSelectionSet (.yarn/$$virtual/@apollo-client-virtual-34e8d4dc43/0/cache/@apollo-client-npm-3.4.11-6b935d0490-9eec5e9cf7.zip/node_modules/@apollo/client/cache/inmemory/writeToStore.js:105:20)
2021-10-01T15:29:39.8014422Z 
```

**Comment by TheNoodleMoose** [comment]
@OzzieOrca I went ahead and made https://jira.cru.org/browse/MPDX-7162

**Comment by OzzieOrca** [comment]
Thanks :)

**Pull request merged**